src/mem3/test/eunit/mem3_reshard_test.erl

```diff
@@ -140,6 +141,56 @@ split_one_shard(#{db1 := Db}) ->
     end)}.
 
 
+% Test to check that shard with high number of purges can be split
+split_shard_with_lots_of_purges(#{db1 := Db}) ->
+    {timeout, ?TIMEOUT, ?_test(begin
+        % Set a low purge infos limit, we are planning of overunning it
```

Comment:
Trivial typo in the comment here, suggest "we are planning to overrun it"

Reply:
Thanks for noticing!